src/docs/content/observability/oci-healthcheck.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "OCI Health Check"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+weight: 0
+tags:
+  - Monitoring
+  - oci_healthcheck
+---
+
+{{% alert style="warning" icon="warning" %}}
+Note that this is **OPTIONAL**.  
+{{% /alert %}}
+
+## Introduction
+
+Monitors the health of IP addresses and hostnames, as measured from geographic vantage points of your choosing, using HTTP and ping probes. After configuring a health check, you can view the monitor's results. The results include the location from which the host was monitored, the availability of the endpoint, and the date and time the test was performed.
+
+## Create OCI HealthCheck
+
+Click Monitoring -> Health Checks -> Create HealthCheck
```

Comment:
~Click~ Navigate to. and use the code block `Monitoring -> Health Checks -> Create HealthCheck` to denote the navigation action